src/components/Notification/Notification.tsx

```diff
@@ -97,6 +101,7 @@ const Notification = ({
   timeout,
   timestamp,
   title,
+  titleElement: Component = "h5",
```

Comment:
Here I feel like renaming the prop makes the render bit slightly more confusing

Reply:
Ah, right. Got it

Reply:
Renamed this to `TitleComponent` just now.